### PR TITLE
Update to the latest `config`

### PR DESCRIPTION
**Pull request by yevhenii-nadtochii**
This PR updates the repository to use the latest `config`.

Mostly, this is done in order to pull in the following changes:

1. [Make published artifacts full-fledged #335](https://github.com/SpineEventEngine/config/pull/335).
2. [Bump dependencies #344](https://github.com/SpineEventEngine/config/pull/344).

Applying of those changes led to the following:

1. `testJar` artifact has disappeared from the list of artifacts published by the modules. This artifact is [optional](https://github.com/SpineEventEngine/config/pull/340) now and not published by default.
2. `protoJar` artifact has also disappeared in modules that don't have any proto definitions.

Please note, the repository does not use the latest version of ErrorProne and some Spine libraries. In particular `mc-java`. Take a look on #176 for details.

The library version is bumped to `2.0.0-SNAPSHOT.67`.

### CODE REVIEW

**Comment by yevhenii-nadtochii** [comment]
@armiol Please take a look.

File: build.gradle.kts [hunk 1]
```diff
@@ -74,37 +74,36 @@ buildscript {
...
-            gitHub("gcloud-java")
-        )
-    }
-    projectsToPublish.addAll(
+    modules = setOf(
         "datastore",
         "stackdriver-trace",
         "testutil-gcloud",
-        "pubsub"
+        "pubsub",
```
Comment (armiol) [inline build.gradle.kts]: Please remove the comma.

File: version.gradle.kts [hunk 2]
```diff
@@ -24,8 +24,9 @@
...
-val spineBaseTypesVersion: String by extra("2.0.0-SNAPSHOT.75")
-val spineCoreVersion: String by extra("2.0.0-SNAPSHOT.89")
-val mcJavaVersion: String by extra("2.0.0-SNAPSHOT.76")
-val versionToPublish: String by extra("2.0.0-SNAPSHOT.66")
+val spineBaseVersion: String by extra("2.0.0-SNAPSHOT.88")
+val spineBaseTypesVersion: String by extra("2.0.0-SNAPSHOT.88")
+val spineCoreVersion: String by extra("2.0.0-SNAPSHOT.94")
+val mcJavaVersion: String by extra("2.0.0-SNAPSHOT.83")
+
+val versionToPublish: String by extra("2.0.0-SNAPSHOT.67")
```
Comment (armiol) [inline version.gradle.kts]: I would go with `94` here, as these artifacts are closely related to `core-java`.

**Review by armiol — APPROVED** [review]
LGTM with a few comments.

**Pull request merged**